src/helpers/activity.js

```diff
@@ -0,0 +1,55 @@
+import { missedCallTypes } from './constants';
+
+export function getActivityIds(activities) {
+    return Object.keys(activities);
+}
+
+export function getMissedCallCount(activities) {
+    let totalMissedActivities = 0;
+
+    getInbox(activities).forEach(activity => {
+        if (activity.call_type && missedCallTypes.includes(activity.call_type)) {
+            totalMissedActivities += 1;
+        }
+    });
+
+    return totalMissedActivities;
```

Comment:
would prefer 

```
getInbox(activities).filter(a => 
  a.call_type && missedCallTypes.includes(activity.call_type)
).length
```

Reply:
Nice, I like that a lot better actually.